Adding 13 new faculty to MBZUAI

PJPurnell opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [X] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [X] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [X] Combine multiple updates to a single institution into a **single PR.**

- [X] Only submit one pull request per institution.

- [X] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [X] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [X] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [X] Check to make sure that you have no spaces after commas, or any missing fields.

- [X] Check to make sure the home page is correct.

- [X] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [X] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [X] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

Commit:
Adding 13 new faculty to MBZUAI

emeryberger commented on csrankings-k.csv:
```diff
@@ -533,6 +533,7 @@ Kensuke Harada,Osaka University,https://www.roboticmanipulation.org,X5hwUlIAAAAJ
 Kent E. Seamons,Brigham Young University,https://cs.byu.edu/faculty/seamons,XY4ky-YAAAAJ
 Kent Quanrud,Purdue University,https://www.kentquanrud.com,RrWxBOAAAAAJ
 Kenta Nakai,University of Tokyo,https://www.hgc.jp/~knakai,glkyhpcAAAAJ
+Kentaro Inui,MBZUAI,https://mbzuai.ac.ae/study/faculty/kentaro-inui,38_o3-kAAAAJ
```
Visiting Professors not eligible for inclusion.

emeryberger commented on csrankings-i.csv:
```diff
@@ -383,7 +383,7 @@ Ivan Hal Sudborough,University of Texas at Dallas,https://explorer.utdallas.edu/
 Ivan I. Garibay,University of Central Florida,https://www.cs.ucf.edu/~garibay,rPuLfoMAAAAJ
 Ivan Kisel,Goethe University Frankfurt,https://www.fias.science/de/computerwissenschaften/gruppen/ivan-kisel,NOSCHOLARPAGE
-Ivan Laptev,Ecole Normale Superieure,https://www.di.ens.fr/~laptev,-9ifK0cAAAAJ
+Ivan Laptev,MBZUAI,https://mbzuai.ac.ae/study/faculty/ivan-laptev,-9ifK0cAAAAJ
```
Still a visiting professor? (per personal web page)

emeryberger commented on csrankings-h.csv:
```diff
@@ -866,6 +866,7 @@ Horst-Michael Groß,TU Ilmenau,https://www.tu-ilmenau.de/neurob/team/prof-dr-hor
 Horácio C. Neto,Universidade de Lisboa,https://tecnico.ulisboa.pt/pt/tag/horacio-neto,17oPwbYAAAAJ
 Hosagrahar Visvesvaraya Jagadish,University of Michigan,http://web.eecs.umich.edu/~jag,SKVnHakAAAAJ
 Hosna Jabbari,University of Victoria,https://onlineacademiccommunity.uvic.ca/cobra,CiBbx70AAAAJ
+Hosni Ghedira,MBZUAI,https://mbzuai.ac.ae/study/faculty/hosni-ghedira,bdv8dsIAAAAJ
```
Professors of Practice not eligible for inclusion

PJPurnell commented:
Hi Emery,


 *   Kentaro Inui: I confirm Kentaro has recently switched to a regular position, and is no longer ‘visiting’. We have now updated our website<https://mbzuai.ac.ae/study/faculty/kentaro-inui/>.
 *   Ivan Laptev: I confirm Ivan has also switched to a regular position. He has now amended the personal website<https://www.di.ens.fr/~laptev/> I believe you are referring to.
 *   Hosni Khedira: Is indeed professor of practice and I shall remove him from the PR.

Is the correct course of action to close my PR and commit a new one?

Best wishes,
Philip



Philip Purnell
Head of Institutional Research and Benchmarking
T: +971 2 811 3226
E: ***@***.***
http://www.mbzuai.ac.ae/<https://mbzuai.ac.ae/>

Mohamed bin Zayed University of Artificial Intelligence
Masdar City, Building 1B, 3rd Floor
Abu Dhabi, UAE

[cid:32_5f0d4623-c70a-4f6f-aa82-dcb8eb00c458.png]
 The world’s first specialized research-based AI university

[cid:bannerforjan2024_aaa8fcd0-fd35-4740-a28c-f68e7694b572.gif]
<https://mbzuai.ac.ae/sustainability/>

From: Emery Berger ***@***.***>
Date: Saturday, 28 September 2024 at 3:20 AM
To: emeryberger/CSrankings ***@***.***>
Cc: Philip Purnell ***@***.***>, Author ***@***.***>
Subject: Re: [emeryberger/CSrankings] Adding 13 new faculty to MBZUAI (PR #7731)

@emeryberger requested changes on this pull request.

________________________________

In csrankings-k.csv<https://cas5-0-urlprotect.trendmicro.com/wis/clicktime/v1/query?url=https%3a%2f%2fare01.safelinks.protection.outlook.com%2f%3furl%3dhttps%253A%252F%252Fgithub.com%252Femeryberger%252FCSrankings%252Fpull%252F7731%2523discussion_r1779292459%26data%3d05%257C02%257Cphilip.purnell%2540mbzuai.ac.ae%257Cc02d4a5ec60f4abc164308dcdf4ae9b9%257Cc93272d31b074b3da3b619b34a973915%257C0%257C0%257C638630760078520351%257CUnknown%257CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%253D%257C0%257C%257C%257C%26sdata%3d9bG%252BbRdtmZs%252Fu6GGhpC%252Bv9CAzvOYPFi06vvM3WnG8MY%253D%26reserved%3d0&umid=c5d82254-0d16-4fcd-8ec1-60aa94c7b9b8&auth=30f123a032afde9cf6a759dbc7bbb78d9968b0f7-86ae38c3f673c0db989365cc9d215449e0c6fcbe>:

> @@ -533,6 +533,7 @@ Kensuke Harada,Osaka University,https://ems-urlprotect.trendmicro.com/wis/clicktime/v1/query?url=https%3a%2f%2fwww.roboticmanipulation.org%2cX5hwUlIAAAAJ&umid=92ebf910-5b50-41ac-abbc-758e254dea53&auth=fcf8495ed58b75d2a2b3723c5458c774642b48aa-7c2494ebbcf838bf23b622c64bc95551ef724c32

 Kent E. Seamons,Brigham Young University,https://cs.byu.edu/faculty/seamons,XY4ky-YAAAAJ

 Kent Quanrud,Purdue University,https://ems-urlprotect.trendmicro.com/wis/clicktime/v1/query?url=https%3a%2f%2fwww.kentquanrud.com%2cRrWxBOAAAAAJ&umid=92ebf910-5b50-41ac-abbc-758e254dea53&auth=fcf8495ed58b75d2a2b3723c5458c774642b48aa-fd3acc3259af256e20d48290f3c142ef73d145b1

 Kenta Nakai,University of Tokyo,https://www.hgc.jp/~knakai,glkyhpcAAAAJ

+Kentaro Inui,MBZUAI,https://mbzuai.ac.ae/study/faculty/kentaro-inui,38_o3-kAAAAJ

Visiting Professors not eligible for inclusion.

________________________________

In csrankings-i.csv<https://cas5-0-urlprotect.trendmicro.com/wis/clicktime/v1/query?url=https%3a%2f%2fare01.safelinks.protection.outlook.com%2f%3furl%3dhttps%253A%252F%252Fgithub.com%252Femeryberger%252FCSrankings%252Fpull%252F7731%2523discussion_r1779293171%26data%3d05%257C02%257Cphilip.purnell%2540mbzuai.ac.ae%257Cc02d4a5ec60f4abc164308dcdf4ae9b9%257Cc93272d31b074b3da3b619b34a973915%257C0%257C0%257C638630760078544766%257CUnknown%257CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%253D%257C0%257C%257C%257C%26sdata%3djo2zjZ5ZKQh0DYFOIP2VMnSqoTNAVwn4YPV4b8DKngQ%253D%26reserved%3d0&umid=c5d82254-0d16-4fcd-8ec1-60aa94c7b9b8&auth=30f123a032afde9cf6a759dbc7bbb78d9968b0f7-805e77638b83265c225ea65d3d527f811a1ec393>:

> @@ -383,7 +383,7 @@ Ivan Hal Sudborough,University of Texas at Dallas,https://explorer.utdallas.edu/

 Ivan Herman,CWI,http://homepages.cwi.nl/~ivan,ofRrImEAAAAJ

 Ivan I. Garibay,University of Central Florida,https://www.cs.ucf.edu/~garibay,rPuLfoMAAAAJ

 Ivan Kisel,Goethe University Frankfurt,https://ems-urlprotect.trendmicro.com/wis/clicktime/v1/query?url=https%3a%2f%2fwww.fias.science%2fde%2fcomputerwissenschaften%2fgruppen%2fivan-kisel%2cNOSCHOLARPAGE&umid=92ebf910-5b50-41ac-abbc-758e254dea53&auth=fcf8495ed58b75d2a2b3723c5458c774642b48aa-e11386a9cb3004b7edfc2093342910f182fe3b23

-Ivan Laptev,Ecole Normale Superieure,https://www.di.ens.fr/~laptev,-9ifK0cAAAAJ

+Ivan Laptev,MBZUAI,https://mbzuai.ac.ae/study/faculty/ivan-laptev,-9ifK0cAAAAJ

Still a visiting professor? (per personal web page)

________________________________

In csrankings-h.csv<https://cas5-0-urlprotect.trendmicro.com/wis/clicktime/v1/query?url=https%3a%2f%2fare01.safelinks.protection.outlook.com%2f%3furl%3dhttps%253A%252F%252Fgithub.com%252Femeryberger%252FCSrankings%252Fpull%252F7731%2523discussion_r1779293528%26data%3d05%257C02%257Cphilip.purnell%2540mbzuai.ac.ae%257Cc02d4a5ec60f4abc164308dcdf4ae9b9%257Cc93272d31b074b3da3b619b34a973915%257C0%257C0%257C638630760078558148%257CUnknown%257CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%253D%257C0%257C%257C%257C%26sdata%3dQDXLR%252BvkXY8MhjG6mXdfTOUrIrjHj%252FYkRsfXsFGAdPk%253D%26reserved%3d0&umid=c5d82254-0d16-4fcd-8ec1-60aa94c7b9b8&auth=30f123a032afde9cf6a759dbc7bbb78d9968b0f7-e535cdde537001c04f25c02a2d3a6ad1bab55851>:

> @@ -866,6 +866,7 @@ Horst-Michael Groß,TU Ilmenau,https://www.tu-ilmenau.de/neurob/team/prof-dr-hor

 Horácio C. Neto,Universidade de Lisboa,https://tecnico.ulisboa.pt/pt/tag/horacio-neto,17oPwbYAAAAJ

 Hosagrahar Visvesvaraya Jagadish,University of Michigan,http://web.eecs.umich.edu/~jag,SKVnHakAAAAJ

 Hosna Jabbari,University of Victoria,https://onlineacademiccommunity.uvic.ca/cobra,CiBbx70AAAAJ

+Hosni Ghedira,MBZUAI,https://mbzuai.ac.ae/study/faculty/hosni-ghedira,bdv8dsIAAAAJ

Professors of Practice not eligible for inclusion

—
Reply to this email directly, view it on GitHub<https://github.com/emeryberger/CSrankings/pull/7731#pullrequestreview-2334792773>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AYKJTJBQML2K2UHP4FS7V4LZYXRTHAVCNFSM6AAAAABO7QEXMWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDGMZUG44TENZXGM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

emeryberger commented:
You can just remove the professor of practice from this PR.

Commit:
Adding twelve new faculty to MBZUAI
